db/gtkaml

```diff
@@ -1,14 +0,0 @@
-R2PM_BEGIN
```

Comment:
dont delete this, i havhe some projects still using it . but yeah, svn sucks :<<< and the project is sadly abandoned :_ but it was awesome, so dont remove it and ill merge